l10n_it_fatturapa_out_riba/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "ITA - Controllo sulla banca ricevente per pagamento RiBa",
+    "summary": """ This module avoids to add a partner bank when a RiBa
+        payment mode is set up on customer invoices.""",
+    "version": "14.0.0.0.0",
```

Comment:
```suggestion
    "version": "14.0.1.0.0",
```
Vedi https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#12version-numbers:
> The version number in the module manifest should be the Odoo major version (e.g. 12.0) followed by the module x.y.z version numbers. For example: 12.0.1.0.0 is expected for the first release of an 12.0 module.

Reply:
:+1: